workspace/utilities/lib/share-link.xsl

```diff
@@ -35,13 +35,16 @@
 		<xsl:param name="separator" select="' - '" />
 		<xsl:param name="url" />
 
+		<xsl:variable name="encoded-status" select="str:encode-uri($status, true())" />
+		<xsl:variable name="encoded-body" select="str:encode-uri($url, true())" />
```

Comment:
why do we need the true() param ?

Reply:
Yes we need true() to encode special characters like "&". If they are not encoded, it will break the query-string. ALso, for any content in the query-string, we want all characters to be encoded

Reply:
:+1: